### ISSUE TITLE
Is it possible to remove the testing/validation error calculation in matconvnet?

### ISSUE THREAD

**Issue by anselmoferreira** [vlfeat/matconvnet#824]
In my imdb I have only imdb.images.set==1 and imdb.images.set==3, which means I am using the testing data as the validation data. I am using the original parameters (learning rate, etc) from the MNIST dataset network.

Because my testing data is too big (1.4M images) and I just want to run the network and save the model to use it later for another application, how can I do to remove the validation error calculation in cnn_train to make it running faster?

**Comment by albanie** [comment]
Hi @anselmoferreira, assuming you are using either `cnn_train` or `cnn_train_dag`, you can pass a `NaN` value to prevent the evaluation of the validation set (this is handled [here](https://github.com/vlfeat/matconvnet/blob/master/examples/cnn_train.m#L54)). So, if you were when using the `mnist` example code, you would simply change [this line](https://github.com/vlfeat/matconvnet/blob/master/examples/mnist/cnn_mnist.m#L58) in `cnn_mnist` to 

    opts.train, ...
    'val', NaN) ;

**Comment by albanie** [comment]
Yep, that's correct.

**Comment by anselmoferreira** [comment]
Hi @albanie ,

I discovered that there are also two other things to do:

1- comment line 151 of cnn_train.m

`%stats.val(epoch) = lastStats.val ;`

2- in line 165 of cnn_train.m replace 

`for f = {'train', 'val'} with `

with 
   ` for f = {'train'}`

3- Replace line 157 of cnn_train.m with

```
 plots = setdiff(...
      fieldnames(stats.train)', ...
      {'num', 'time'}) ;
```
Now it is working.

Bests,

**Comment by albanie** [comment]
Great, thanks!
